packages/qdrant-loader/src/qdrant_loader/connectors/confluence/connector.py

```
@@ -792,162 +804,249 @@ def _clean_html(self, html: str) -> str:
-            while True:
-                try:
-                    page_count += 1
-                    logger.debug(
-                        f"Fetching page {page_count} of Confluence content (start={start})"
-                    )
-                    response = await self._get_space_content_datacenter(start)
-                    results = response.get("results", [])
+            except Exception as e:
+                logger.error(
+                    f"Failed to fetch content from space {self.config.space_key}: {e!s}"
+                )
```

Ruff (0.15.15)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)